Refine DB test (#1)
scalar-labs/kelpie-test: scalardb-test/README.md

> # Benchmark/Verification Test for Scalar DB

Reviewer: ```suggestion
# Benchmark and verification for Scalar DB
```

## Usage
1. Set up an environment
    - This job requires a client to execute this job and Cassandra cluster or Cosmos DB which can be accessed from the client

2. Build
    ```console
    $ gradle shadowJar
    ```

3. Make your config file based on `benchmark-config.toml` or `verification-config.toml`

> 4. Kelpie binary (zip) from https://github.com/scalar-labs/kelpie/releases/ and unzip

Reviewer: ```suggestion
4. Download Kelpie binary (zip) from https://github.com/scalar-labs/kelpie/releases/ and unzip it
```

    - You can also build Kelpie from the source

5. Execute a job
    ```console
    $ ${KELPIE}/bin/kelpie --config your_config.toml
    ```

## Workload

> - Transaction

Reviewer: ```suggestion
- Payment transaction
```

  - Transfer money between randomly selected 2 accounts
    1. Start a transaction
    2. Read 2 accounts
    3. Calculate new balances

>     4. Update 2 accounts with new balances

Reviewer: ```suggestion
    4. Update 2 accounts with the new balances
```

    5. Commit the transaction

- Account
  - An account has 2 types `0` and `1`
    - Basically, the type `0` of an account is used
    - When the source account ID and the destination account ID is the same, the money will be sent from type `0` to `1` of the account ID
  - The number of accounts can be set at `[test_config]` `num_accounts` in your config file

## Cassandra Killer

> - Cassandra Killer is an injector to make nodes crash randomly and restart the nodes

Reviewer: ```suggestion
- Cassandra Killer is an injector to kill nodes randomly and restart the nodes
```

>   - The nodes to be crashed are selected randomly from `[killer_config]` `contact_points`

Reviewer: ```suggestion
  - The nodes to be killed are selected randomly from `[killer_config]` `contact_points`
```
